chains.yaml

```diff
@@ -545,10 +545,23 @@
   github-organization: onomyprotocol
   github-repo: onomy
   dockerfile: cosmos
+  # TODO: Heighliner will not produce working builds until https://github.com/onomyprotocol/onomy/pull/129 merged.
+  # If PR not merged for your git-ref, then update and push to our fork, https://github.com/strangelove-ventures/onomy, with Onomy upstream.
+  # And manually run heighliner command with `-o strangelove-ventures --git-ref main --tag <semver>`.
   build-target: make install
```

Comment:
@agouin Decided to take this approach vs. a long build target. 

As of this writing, https://github.com/onomyprotocol/onomy/pull/129 has no activity from the Onomy team. I've reached out to them in our dedicated slack channel.